Warewulfd panic after adding 2nd node to the cluster configuration

Used latest version 4.3.0-1.git_235c23c

Problem descriptions.
while adding a 2nd node to the cluster using wwctl command the warewulfd panic.
Entries in the nodes.conf are made but warewulfd failed silent. 
Restart failed as well, without any error message on the console.
Only a status request will show the problem (need to have an error message for the user).
Restarts are not possible unless the new added node in the nodes.conf are deleted
After deleting the new added node in the nodes.conf warewuld will start normal and the 1st node in the cluster can boot.

start of console log #################
[root@emil log]0 # wwctl node add hpc3 --ipaddr 10.25.121.13  --discoverable
Added node: hpc3
[root@emil log]0 # systemctl status  warewulfd
● warewulfd.service - Warewulf cluster management daemon
   Loaded: loaded (/usr/lib/systemd/system/warewulfd.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Fri 2022-06-24 15:32:20 CEST; 15s ago
     Docs: https://warewulf.hpcng.org/
  Process: 1142324 ExecStart=/usr/bin/wwctl server start (code=exited, status=2)
 Main PID: 1142324 (code=exited, status=2)

Jun 24 15:32:20 emil.k-net.ka.de systemd[1]: warewulfd.service: Service RestartSec=100ms expired, scheduling restart.
Jun 24 15:32:20 emil.k-net.ka.de systemd[1]: warewulfd.service: Scheduled restart job, restart counter is at 6.
Jun 24 15:32:20 emil.k-net.ka.de systemd[1]: Stopped Warewulf cluster management daemon.
Jun 24 15:32:20 emil.k-net.ka.de systemd[1]: warewulfd.service: Start request repeated too quickly.
Jun 24 15:32:20 emil.k-net.ka.de systemd[1]: warewulfd.service: Failed with result 'exit-code'.
Jun 24 15:32:20 emil.k-net.ka.de systemd[1]: Failed to start Warewulf cluster management daemon.
[root@emil log]3 # systemctl start  warewulfd
[root@emil log]0 # systemctl status  warewulfd
● warewulfd.service - Warewulf cluster management daemon
   Loaded: loaded (/usr/lib/systemd/system/warewulfd.service; enabled; vendor preset: disabled)
   Active: failed (Result: exit-code) since Fri 2022-06-24 15:32:47 CEST; 10s ago
     Docs: https://warewulf.hpcng.org/
  Process: 1142586 ExecStart=/usr/bin/wwctl server start (code=exited, status=2)
 Main PID: 1142586 (code=exited, status=2)

Jun 24 15:32:47 emil.k-net.ka.de systemd[1]: warewulfd.service: Service RestartSec=100ms expired, scheduling restart.
Jun 24 15:32:47 emil.k-net.ka.de systemd[1]: warewulfd.service: Scheduled restart job, restart counter is at 5.
Jun 24 15:32:47 emil.k-net.ka.de systemd[1]: Stopped Warewulf cluster management daemon.
Jun 24 15:32:47 emil.k-net.ka.de systemd[1]: warewulfd.service: Start request repeated too quickly.
Jun 24 15:32:47 emil.k-net.ka.de systemd[1]: warewulfd.service: Failed with result 'exit-code'.
Jun 24 15:32:47 emil.k-net.ka.de systemd[1]: Failed to start Warewulf cluster management daemon.

end of console log############

it seems that the entry in the nodes.conf is ok but warewulf failed to use them
"nodes.conf Extract "
  hpc3:
    discoverable: "true"
    profiles:
    - default
    network devices:
      default:
        onboot: "true"
        ipaddr: 10.25.121.13


[root@emil log]0 # cat warewulfd.log
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00069f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00069f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00069f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00069f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00061f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00069f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00025f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00061f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00069f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/pkg/node.(*nodeYaml).FindAllNodes(0xc00061f338)
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/node/constructors.go:262 +0x43f1
github.com/hpcng/warewulf/internal/pkg/warewulfd.loadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:36 +0x56
github.com/hpcng/warewulf/internal/pkg/warewulfd.LoadNodeDB()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/nodedb.go:25 +0x6d
github.com/hpcng/warewulf/internal/pkg/warewulfd.RunServer()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/warewulfd.go:42 +0xd9
github.com/hpcng/warewulf/internal/pkg/warewulfd.DaemonStart()
        /builddir/build/BUILD/warewulf-4.3.0/internal/pkg/warewulfd/daemon.go:74 +0x65
github.com/hpcng/warewulf/internal/app/wwctl/server/start.CobraRunE(0x15ecb40, {0xd420cb, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/server/start/main.go:20 +0xfd
github.com/spf13/cobra.(*Command).execute(0x15ecb40, {0x1691d20, 0x0, 0x0})
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
        /builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25

--- a/internal/pkg/node/constructors.go
+++ b/internal/pkg/node/constructors.go
@@ -256,6 +256,9 @@ func (config *nodeYaml) FindAllNodes() ([]NodeInfo, error) {
 				n.NetDevs[devname].OnBoot.SetAlt(netdev.OnBoot, p)
 				n.NetDevs[devname].Primary.SetAlt(netdev.Primary, p)
 				if len(netdev.Tags) != 0 {
+					if len(n.NetDevs[devname].Tags) == 0 {
+						n.NetDevs[devname].Tags = make(map[string]*Entry)
+					}
 					for keyname, key := range netdev.Tags {
 						if _, ok := n.NetDevs[devname].Tags[keyname]; !ok {
 							var keyVar Entry

